Gemfile.lock

```diff
@@ -78,17 +78,17 @@ GEM
       regexp_parser (~> 1.5)
       xpath (~> 3.2)
     childprocess (3.0.0)
-    concurrent-ruby (1.1.5)
-    crass (1.0.5)
+    concurrent-ruby (1.1.6)
```

Comment:
### [concurrent-ruby](http://www.concurrent-ruby.com)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.5 → 1.1.6

[_[change-log](https://github.com/ruby-concurrency/concurrent-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/ruby-concurrency/concurrent-ruby)_]





<details>
<summary>Commits</summary>

A change of **113** commits. See the full changes on [the compare page](https://github.com/ruby-concurrency/concurrent-ruby/compare/v1.1.5...v1.1.6).

These are the first 10 commits:
- (3073d2d) [Add a metadata link to changelog](https://github.com/ruby-concurrency/concurrent-ruby/commit/3073d2de6298d1e7187b698ccae1a1676170289e)
- (e411434) [Merge pull request #805 from viraptor/patch-1](https://github.com/ruby-concurrency/concurrent-ruby/commit/e4114347cfebf8521ed0360d7fb1e7889f0a4733)
- (a423377) [Missing require for releaseing](https://github.com/ruby-concurrency/concurrent-ruby/commit/a423377c8ec16bb2051bec483a0438165162c5a6)
- (8ad788d) [Fix spec timing](https://github.com/ruby-concurrency/concurrent-ruby/commit/8ad788d393000a1724f39c85a11b40b1478c1b23)
- (8344536) [CI: Use 2.5.5, 2.6.2](https://github.com/ruby-concurrency/concurrent-ruby/commit/8344536368a3de6f173595d5509ab852710111aa)
- (addc943) [More details in final report](https://github.com/ruby-concurrency/concurrent-ruby/commit/addc9430354019504de8cb68a6d7c43a0a7981aa)
- (bae1761) [Include intermediate report into final](https://github.com/ruby-concurrency/concurrent-ruby/commit/bae1761b7195b492a572683ade6d08b83f7c5afe)
- (38d5c81) [CI: jruby-9.2.7.0](https://github.com/ruby-concurrency/concurrent-ruby/commit/38d5c81b4e09e732975aa266842fd8427d805d4d)
- (daeafed) [CI: Use 2.4.6](https://github.com/ruby-concurrency/concurrent-ruby/commit/daeafed95e9f186fb48f8dc322fbf6cda1cdaff7)
- (d11b29c) [Merge pull request #806 from olleolleolle/patch-1](https://github.com/ruby-concurrency/concurrent-ruby/commit/d11b29c37b81320ca7126cc9cd85f4f3d17a78a3)


</details>